cg_hermes/constants/tags.py

```diff
@@ -0,0 +1,637 @@
+from enum import StrEnum
+from typing import Any
+
+
+class AlignmentTags(StrEnum):
```

Comment:
Does it make any sense to split these tags into different classes or should the following ones be merge into one as they function in the same way?

    AlignmentTags,
    AnalysisTags,
    BioinfoToolsTags,
    FamilyTags,
    QCTags,
    RawDataTags,
    ReportTags,
    VariantTags,
    BalsamicTags,
    MicrosaltTags,
    MipTags,
    MutantTags,
    NextflowTags,
    RnafusionTags,
    TaxprofilerTags,
    TomteTags,

Reply:
It does make sense if we want to display it using github pages: https://clinical-genomics.github.io/hermes/.

I think we do, but for some reason the automatic action is not triggered anymore 🤔 I changed some options in the admin view but I'm not sure if it will be triggered

Reply:
wow, first time I see that github page, but as you said it doesn't seem to be updated. Not sure how useful that page is, to be honest. If we can get a cleaner implementation by dropping these pages I would argue that it's worth it. That being said, I think it's fair to leave it for now and do it in a different PR if needed

Reply:
Yep, maybe we can just switch to markdown for these tables and centralise them. Sounds like a good idea 👍 . I will open an issue